packages/nodes-base/nodes/Transform/utils/utils.ts

```diff
@@ -8,7 +10,8 @@ export const prepareFieldsArray = (fields: string | string[], fieldName = 'Field
 	if (Array.isArray(fields)) {
 		return fields;
 	}
-	throw new Error(
+	throw new ApplicationError(
```

Comment:
I think this change does not belong here right?

Reply:
Addressing here: https://github.com/n8n-io/n8n/pull/7966